src/hotspot/share/opto/idealGraphPrinter.hpp

```diff
@@ -92,6 +92,7 @@ class IdealGraphPrinter : public CHeapObj<mtCompiler> {
   PhaseChaitin* _chaitin;
   bool _traverse_outs;
   Compile *C;
+  double max_freq;
```

Comment:
Fields should have a leading underscore.

Reply:
Done.